lib/private/Config/UserConfig.php

```diff
@@ -1843,7 +1844,9 @@ private function matchAndApplyLexiconDefinition(
 		}
 
 		$lazy = $configValue->isLazy();
-		$default = $configValue->getDefault() ?? $default; // default from Lexicon got priority
+		// default from Lexicon got priority but it can still be overwritten by admin
+		// with 'lexicon.default.userconfig' => [<app>.<key> => 'my value'], in config.php
```

Comment:
```suggestion
		// with 'lexicon.default.userconfig' => [<app> => [<key> => 'my value']], in config.php
```
Better nested arrays than magic string concatenation?

Reply:
could be, mine seems just more human readable